app/lib/medical-transcription-analysis-stack.ts

```diff
@@ -281,13 +282,54 @@ export class MedicalTranscriptionAnalysisStack extends cdk.Stack {
             actions: ["transcribe:StartStreamTranscriptionWebSocket","transcribe:StartMedicalStreamTranscription","comprehendmedical:InferICD10CM","comprehendmedical:InferRxNorm","comprehendmedical:DetectEntitiesV2"]
           })
         );
+        
+        // Dynamodb
+        const TableSessions = new ddb.Table(this, 'Table1', {
+          tableName: 'Sessions',
+          partitionKey: { name: 'PatientId', type: ddb.AttributeType.STRING },
+          sortKey: {name: 'SessionId', type: ddb.AttributeType.NUMBER },
+          serverSideEncryption: true
+        });
+
+        TableSessions.addGlobalSecondaryIndex({
+          indexName: "hcpIndex",
+          partitionKey: { name: 'HealthCareProId', type: ddb.AttributeType.STRING },
```

Comment:
Can we use HealthCareProfessionalId as the key name? Lets avoid introducing non-standard acronyms